newrelic/api/ml_model.py

```diff
@@ -52,3 +53,35 @@ def get_ai_message_ids(response_id=None):
         return [{"conversation_id": conversation_id, "request_id": request_id, "message_id": _id} for _id in ids]
     warnings.warn("No message ids found. get_ai_message_ids must be called within the scope of a transaction.")
     return []
+
+
+def record_ai_feedback_event(
+    message_id, rating, conversation_id=None, request_id=None, category=None, message=None, metadata=None
+):
+    transaction = current_transaction()
+    if not transaction:
+        warnings.warn(
+            "No message feedback events will be recorded. record_ai_feedback must be called within the "
+            "scope of a transaction."
+        )
+        return
+
+    feedback_message_id = str(uuid.uuid4())
+    try:
+        rating = str(rating)
```

Comment:
This doesn't seem necessary to me but I could be missing something. Why are we casting this to a string?

Reply:
The thought was that the rating could be a string or int (numeric rating) so this would cast the rating to always be the same type. Given that it's only being casted for the sake of adding it to the feedback event and not touched further, we can keep the original type in tact and remove this.